setup.py

```diff
@@ -343,7 +343,7 @@ def run(self):
       keywords='Machine Learning',
       author='Ryosuke Fukatani',
```

Comment:
How about change to 'RGF-Team'?

Reply:
OK.

Reply:
BTW, let's reach the agreement: `RGF-Team` or `RGF-team`?
COPYING has `RGF-team` (literally from GitHub), but it's not hard to change.

Reply:
Ah, sorry. I meaned `RGF-team`.

Reply:
Thanks! Done.